modules/nf-core/gatk4/printsvevidence/main.nf

```diff
@@ -8,28 +8,28 @@ process GATK4_PRINTSVEVIDENCE {
         'quay.io/biocontainers/gatk4:4.2.6.1--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(evidence), path(evidence_index)
+    tuple path(evidence_files), path(evidence_indices)
     path bed
     path fasta
     path fasta_fai
     path dict
 
-
     output:
-    tuple val(meta), path("*.txt.gz")       , emit: printed_evidence
-    tuple val(meta), path("*.txt.gz.tbi")   , emit: printed_evidence_index
-    path "versions.yml"                     , emit: versions
+    path "*.txt.gz"       , emit: printed_evidence
+    path "*.txt.gz.tbi"   , emit: printed_evidence_index
+    path "versions.yml"   , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def prefix = task.ext.prefix ?: "printsvevidence"
     def intervals = bed ? "--intervals ${bed}" : ""
     def reference = fasta ? "--reference ${fasta}" : ""
+    def input_files = evidence_files.collect({"--evidence-file $it"})join(' ')
```

Comment:
```suggestion
    def input_files = evidence_files.collect({"--evidence-file $it"}).join(' ')
```
? probably missing a dot here?

Reply:
Oh but it works lol. I've been noticing that lately that I miss them a couple times, but the code keeps working